Makefile

```diff
@@ -0,0 +1,9 @@
+NAME	:= gotestsum
+SRC_EXT	:= gz
+VERSION	:= 0.4.0
```

Comment:
Shouldn't need this.  It will be extracted from the specfile automatically.

Reply:
Nifty, hadn't realized that the included makefile handled all this stuff.